src/Utils/__tests__/resized.jest.ts

```diff
@@ -64,3 +64,39 @@ describe("#resized", () => {
     expect(srcSet).toContain("quality=80")
   })
 })
+
+describe("maxWidthByArea", () => {
```

Comment:
I can't say that I've seen tests like these before. Would you call them unit specs specifically for UI elements?

Reply:
or is it better to say that this is a unit test specifically testing the output of the `maxWidthByArea` function?

Reply:
Just a unit test for a pure function.